builds/creating-build-inputs.adoc

```diff
@@ -3,6 +3,7 @@
 
 :context: creating-build-inputs
 [id='creating-build-inputs_{context}']
+include::modules/common-attributes.adoc[]
```

Comment:
This doesn't quite work here? Where should it be located in the assembly module?

Reply:
^ fixed